dpgen/generator/run.py

```diff
@@ -2469,6 +2467,20 @@ def _make_fp_vasp_configs(iter_index,
         cur_job = json.load(open(os.path.join(modd_path, 'cur_job.json'), 'r'))
         if 'task_min' in cur_job :
             task_min = cur_job['task_min']
+    else:
+        cur_job = {}
+    # support iteration dependent trust levels
+    v_trust_lo = cur_job.get('model_devi_v_trust_lo', jdata.get('model_devi_v_trust_lo', 1e10))
+    v_trust_hi = cur_job.get('model_devi_v_trust_hi', jdata.get('model_devi_v_trust_hi', 1e10))
+    if cur_job.get('model_devi_f_trust_lo'):
```

Comment:
This line will not work if one sets it to zero.

You may write as `if xx is not None`.